src/Umbraco.Infrastructure/Services/Implement/PackagingService.cs

```diff
@@ -155,7 +183,8 @@ public async Task<PagedModel<PackageDefinition>> GetCreatedPackagesAsync(int ski
         using ICoreScope scope = _coreScopeProvider.CreateCoreScope(autoComplete: true);
         PackageDefinition[] packages = _createdPackages.GetAll().WhereNotNull().ToArray();
         var pagedModel = new PagedModel<PackageDefinition>(packages.Length, packages.Skip(skip).Take(take));
-        return await Task.FromResult(pagedModel);
```

Comment:
I noticed we have quite a few occurrences of `await Task.FromResult()` spread throughout the codebase, which seems wasteful, as it wraps the result and adds an additional await (with all the extra async state machine code that's generated by the compiler)...